src/widgetastic_patternfly4/popover.py

```diff
@@ -0,0 +1,68 @@
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import Widget
+
+
+class BasePopover:
+    """Represents a popover.
+    https://www.patternfly.org/v4/components/popover/
+
+    For the popover to work, the base view should include a property
+    popover = Popover()
+    As there is supposed to only be one popover active at a time, this property
+    should work, interacting with the active popover.
+    """
+
+    TITLE = """.//*[self::h1 or self::h2 or self::h3 or self::h4 or self::h5 or self::h6
+                and contains(@class, "pf-c-title")]"""
+    BODY = './/div[contains(@class, "pf-c-popover__body")]'
+    FOOTER = ".//footer[contains(@class, 'pf-c-popover__footer')]"
+    CLOSE = ".//button[@aria-label='Close']"
+
+    def read(self):
+        """Returns the text of the body of the popover as a string."""
+        return self.body
```

Comment:
Do we need to open the popover before reading it?

Reply:
There can be only one popover displayed on a page at a time. My intended usage is that the base view (DriftBaseView) has a `popover = Popover()` component and in a particular view, I rewrite that with something like `popover = CustomPopover()`

There is no component that explicitly triggers a popover. It pops up on the screen when a button is clicked, but this button doesn't ([per pf4 docs](https://www.patternfly.org/v4/components/popover)) have any indication that it opens up a popover. 
So my reasoning was that it would be up to the user (the automation) to open the popover, because it can be triggered from 'random' components. 

Maybe it would be better if I added an `is_displayed` check on some methods, so that they don't throw an exception if the popover is not displayed?

Alternatively, we can impose a "trigger_element` (or similar) as a constructor parameter of Popover.

(note that the triggering element is not within the hiearchy of the popover, instead it is a direct child of <body>)

Reply:
Thank you for the detailed reply!
I think it'd be good to implement both suggestions -- but they could be follow-ups and I wouldn't consider them necessary for this PR.

> Maybe it would be better if I added an is_displayed check on some methods, so that they don't throw an exception if the popover is not displayed?
I think this is a good idea, as long as the user is aware of the reason that the text failed to be read from the popover.
> Alternatively, we can impose a "trigger_element` (or similar) as a constructor parameter of Popover.
We could make such an element optional and define an `open` method that will use the trigger element only if it is defined.